src/utils/CanvasTemplates.js

```diff
@@ -490,4 +490,143 @@ module.exports = class CanvasTemplates {
 
     return encoder.out.getData()
   }
+
+  static async weather ({ t }, title, weather) {
+    const WIDTH = 430
+    const HEIGHT = 350
+
+    const CARD_WIDTH = 383
+    const CARD_HEIGHT = 264
+    const CARD_X_MARGIN = (WIDTH - CARD_WIDTH) * 0.5
+    const CARD_Y_MARGIN = (HEIGHT - CARD_HEIGHT) * 0.5
+
+    const FONTS = (() => {
+      const MEME = Math.random() > 0.99 && '"Comic Sans MS"'
+      const EXTRABOLD = MEME || '"Montserrat ExtraBold"'
+      const REGULAR = MEME || '"Montserrat"'
+      const LIGHT = MEME || '"Montserrat Light"'
+      return {
+        TITLE: `17px ${EXTRABOLD}`,
+        TEMPERATURE: `bold 90px ${EXTRABOLD}`,
+        INFORMATIONS: `17px ${LIGHT}`,
+        WEEK_DAYS: `17px ${REGULAR}`,
+        WEEK_TEMPERATURES: `bold 29px ${LIGHT}`
+      }
+    })()
+
+    const IMAGE_ASSETS = Promise.all([
+      Image.from(Constants.DEFAULT_BACKGROUND_GRAY_PNG, true),
+      Image.buffer(Constants.ARROW_UP_SVG, true),
+      Image.buffer(Constants.WIND_SVG, true)
+    ])
+
+    const WEATHER_ICONS = Promise.all([
+      Image.buffer(Constants.WEATHER_CLEAR_DAY, true),
+      Image.buffer(Constants.WEATHER_CLEAR_NIGHT, true),
+      Image.buffer(Constants.WEATHER_CLOUDY, true),
+      Image.buffer(Constants.WEATHER_FOG, true),
+      Image.buffer(Constants.WEATHER_PARTLY_CLOUDY_DAY, true),
+      Image.buffer(Constants.WEATHER_PARTLY_CLOUDY_NIGHT, true),
+      Image.buffer(Constants.WEATHER_RAIN, true),
+      Image.buffer(Constants.WEATHER_SNOW, true),
+      Image.buffer(Constants.WEATHER_THUNDERSTORM, true),
+      Image.buffer(Constants.WEATHER_WIND, true)
+    ])
```

Comment:
I don't really like the idea of loading every single weather icon even without using some of them in the context.